content/uc-doc/upgrade/upgrade_from_wazo_18.03.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Upgrading from Wazo 18.03
+---
+
+## Rationale
+
+Wazo 18.03 was the last version before a big gap in Wazo versions: the next
+available version is Wazo 19.13. As a consequence, a lot of Wazo installation
```

Comment:
Upgrading (without wazo-dist-upgrade) will go to 19.12
```suggestion
available version is Wazo 19.12. As a consequence, a lot of Wazo installation
```

Reply:
Fixed, thanks.